scripts/common_startup.sh

```diff
@@ -241,16 +241,16 @@ fi
 # Build client if necessary.
 if [ $SKIP_CLIENT_BUILD -eq 0 ]; then
     # Ensure dependencies are installed
-    if [ -n "$VIRTUAL_ENV" ]; then
-        if ! in_venv "$(command -v yarn)"; then
-            echo "Installing yarn into $VIRTUAL_ENV with npm."
-            npm install --global yarn
-        fi
-    elif [ -n "$CONDA_DEFAULT_ENV" ] && [ -n "$CONDA_EXE" ]; then
+    if [ -n "$CONDA_DEFAULT_ENV" ] && [ -n "$CONDA_EXE" ]; then
         if ! in_conda_env "$(command -v yarn)"; then
             echo "Installing yarn into '$CONDA_DEFAULT_ENV' Conda environment with conda."
             $CONDA_EXE install --yes --override-channels --channel conda-forge --name "$CONDA_DEFAULT_ENV" yarn
```

Comment:
The macOS build, which uses conda, fails with:
```
Unknown Syntax Error: Unsupported option name ("--check-files").
```
which it's due to the fact that yarn 3 is installed instead of yarn "classic".

```suggestion
            $CONDA_EXE install --yes --override-channels --channel conda-forge --name "$CONDA_DEFAULT_ENV" 'yarn<2'
```